lib/zuora/objects/product.rb

```diff
@@ -5,7 +5,7 @@ class Product < Base
     validates_inclusion_of :category, :in => ['DefaultCatalog'], :allow_nil => true
     validates_length_of :description, :maximum => 500, :allow_nil => true
 
-    validates_date_of :effective_start_date, :effective_end_date
+    # validates_date_of :effective_start_date, :effective_end_date
```

Comment:
Why is this commented out?

Reply:
Resolved here [04f281f](https://github.com/Sage/zuora/pull/18/commits/04f281f9a0a0b07961e1b1f4ebd5465beb8c27bc)